Northstar.Custom/mod/scripts/vscripts/sh_damage_types.nut

```diff
@@ -773,6 +773,6 @@ void function ReceiveNewDamageSourceIDs( array<string> args )
 {
 	// IDs are inserted to the custom list in triplets, so we can trust these indices exist and the loop will end properly
 	for ( int i = 0; i < args.len(); i += 3 )
-		RegisterWeaponDamageSourceInternal( args[ i ].tointeger(), args[ i + 1 ], StringReplace( args[ i + 2 ], MESSAGE_SPACE_PADDING, " " ) )
+		RegisterWeaponDamageSourceInternal( args[ i ].tointeger(), args[ i + 1 ], StringReplace( args[ i + 2 ], MESSAGE_SPACE_PADDING, " ", true ) )
```

Comment:
This shouldn't have an effect on existing mods, right?

Reply:
It should have no effect. There's nothing that can depend on this system not replacing all spaces because it's all handled internally. I.e. if someone _did_ use more than 1 space, it would break unless they modified the file themselves (in which case compat is unaffected).

I imagine most mods use localization strings though which don't even have spaces until localized.